tools/hangar/__snapshots__/invalid.ts.snap

```diff
@@ -144,6 +144,30 @@ exports[`capture_reassignable.w > stderr 1`] = `
 "
 `;
 
+exports[`class.w > stderr 1`] = `
+"error: \\"x\\" is not initialized
+  --> ../../../examples/tests/invalid/class.w:2:4
+  |
+2 |    x:num;
+  |    ^ \\"x\\" is not initialized
```

Comment:
I like how the error message is next to where the field is defined 👍

Reply:
Me too 😀